src/test/testType.ts

```diff
@@ -151,4 +153,14 @@ export class TestTypeImpl {
   }
 }
 
+function throwIfRunningInsideJest() {
+  if (process.env.JEST_WORKER_ID) {
+    throw new Error(
+        `Playwright Test needs to be invoked via 'npx playwright test' and excluded from Jest test runs.\n` +
+        `Creating one directory for Playwright tests and one for Jest is the recommended way of doing it.\n` +
+        `See https://playwright.dev/docs/test-intro/ for more information about Playwright Test.`,
```

Comment:
https://playwright.dev/docs/test-intro/ is now https://playwright.dev/docs/intro/